Backpack/Carousel/Classes/BPKCarousel.swift

```diff
@@ -17,41 +17,40 @@
  */
 
 import Foundation
+import Backpack_Common
 
 public protocol BPKCarouselDelegate: AnyObject {
     func onImageChange(index: Int)
 }
 
 public final class BPKCarousel: UIView {
```

Comment:
This is now essentially just calling the internal carousel component and handling the PageIndicator, same as the SwiftUI one!